tools/devworkspace-generator/package.json

```diff
@@ -41,6 +41,7 @@
   "homepage": "https://github.com/eclipse-che/che-devfile-registry#readme",
   "dependencies": {
     "@devfile/api": "2.2.1-alpha-1667236163",
```

Comment:
the version of devfile/api library will be updated as a result of https://github.com/eclipse/che/issues/22658